pkg/lib/cockpit-components-listing.jsx

```diff
@@ -304,34 +310,34 @@ var Listing = React.createClass({
             bodyClasses.push("listing-ct-wide");
         var headerClasses;
         var headerRow;
-        var selectableRows;
+        // var selectableRows;
         if (!this.props.children || this.props.children.length === 0) {
             headerClasses = "listing-ct-empty";
             headerRow = <tr><td>{this.props.emptyCaption}</td></tr>;
         } else if (this.props.columnTitles.length) {
+            /* probably not needed, ListingRow.selected prop should be set to 'false' by default
```

Comment:
Keeping it here for simpler revert if I'm wrong.

Will be removed later if no issue found.